molgenis/molgenis-emx2
# feat: allow up/download table CSV by name or id

demo:
- I can upload csv and excel files containing a table, where table name can be case and space agnostic. 


Following request of @YpeZ, uploading a CSV file for one table when having spaces in the name it fails:

![image](https://github.com/molgenis/molgenis-emx2/assets/120060/ff93369f-25dd-4167-94f2-f8b45f4027b2)

In current version for some reason we assume CSV and Excel up/downloads to use the 'id' of the table.
That is silly because we typically upload the csv files from a data.zip and then those use 'name' of the table
So this feature makes it possible to use both 'name' and 'id' (with us prefering to use the 'name').

how to test:
- upload table like 'Collection events.csv' and 'CollectionEvents.csv' should both work
- discussion: make it case insensitive too?
